tensorflow/compiler/xla/service/algebraic_simplifier_test.cc

```diff
@@ -5809,5 +5810,235 @@ TEST_F(AlgebraicSimplifierTest, SliceOfConcat) {
               GmockMatch(m::Parameter(1)));
 }
 
+TEST_F(AlgebraicSimplifierTest, SqrtOfSelfMultiply) {
```

Comment:
Would you please simplify the names used in the HLO instructions for the tests you added, to make it more readable? See CopyReshape for an example.

Reply:
Done